packages/gatsby-theme-docs/src/layouts/internals/layout-release-note.js

```diff
@@ -59,7 +66,9 @@ ReleaseNote.propTypes = {
   description: PropTypes.string.isRequired,
   type: PropTypes.oneOf(['feature', 'enhancement', 'fix']).isRequired,
   topics: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
-  body: PropTypes.string.isRequired,
+  body: PropTypes.string,
+  rawExcerpt: PropTypes.string,
```

Comment:
This is not ideal, but since the GraphQL data source is already strongly typed I think it's acceptable.  We could write a custom validation function to make sure either one is passed if we think this component will be re-used a lot.

Reply:
Hmm but both are required fields, aren't they?

I would expect the logic to be (pseudo-code)

```
if hasMore
  render excerpt
  render read more
else
  render body
```

Reply:
The excerpt is going through a different renderer now than the full body, so in the list view it's always render excerpt and the "If" just covers whether to show the link or not.  

If the body is made required, we would pump up the data size of the page query of the list view with all the unused bodies.